src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__top">
+        <img
+          src="images/imac.jpeg"
+          class="card__top-img"
+          alt="Apptle Monoblock"
+        />
+      </div>
+
+      <div class="card__bottom">
+        <h2 class="card__bottom-tittle">
```

Comment:
There is a typo in the class name `card__bottom-tittle`. It should be `card__bottom-title` to follow the BEM naming convention.